index.js

```diff
@@ -94,12 +126,9 @@ app.post('/api/message', (req, res) => {
   // send payload to Conversation and return result
   assistant.message(payload, (err, data) => {
     if (err) {
-      console.log(err);
```

Comment:
you probably want to keep this in case we need to debug the app.
if you are getting eslint issues just add `//eslint-disable-next-line`